examples/gqlgen-service/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang
+
+WORKDIR /go/src/app
+
+COPY . .
+RUN go mod init gqlen-server
```

Comment:
@suessflorian I'd make these submodules of bramble (github.com/movio/bramble/examples/gqlgen-server) and instantiate them outside of docker. Just so the behaviour and dependencies are the same when running inside of docker and out.

Reply:
That will make any dependencies in the example services be dependencies of `bramble`, even if they are not necessary?

Reply:
I would do the build outside of the docker image if possible, the cache interactions with doing the build in the image slow things down a lot if you are changing the code.

Reply:
Isn't that the case now?  These examples are going up the tree to bramble's `go.mod` and so their dependencies get injected. This should do the reverse of that.

If I do this:

```
cd examples/gqlgen-service && go mod init github.com/movio/bramble/examples/gqlgen && go mod tidy
cd ../../ && go mod tidy
```

It drops dependencies from bramble's main `go.mod` file.

Reply:
By the way, by currently I mean before this PR. I just think we should move the `go mod init` to outside docker and commit the `go.mod` and `go.sum` files.